### DISCUSSION TITLE
directus_sync pull pulls every collection we have, instead of just flows, like the configuration expects

### DISCUSSION THREAD

**Discussion by Dithilli** [tractr/directus-sync#96] [General]
Hi there, 

We're trying to use directus_sync to manage our flows, and when I do directus_sync pull, i get not just the flows collection, but the dozens and dozens of other collections  we have that aren't directus specific. My configuration is setup 

// ./directus-sync.config.js
module.exports = {
  extends: ['./directus-sync.config.base.js'],
  debug: true,
  directusUrl: 'http://localhost:8080/',
  // directusToken: 'my-directus-token',
  directusConfig: {
    clientOptions: {},  // see https://docs.directus.io/guides/sdk/getting-started.html#polyfilling
    restConfig: {}, // see https://docs.directus.io/packages/@directus/sdk/rest/interfaces/RestConfig.html
  },
  dumpPath: './directus-sync',
  collectionsPath: 'collections',
  onlyCollections: ['flows'],
  excludeCollections: ['settings'],
  preserveIds: 'all', // can be '*' or 'all' to preserve all ids, or an array of collections
  snapshotPath: 'snapshot',
  snapshot: true,
  split: true,
  specsPath: 'specs',
  specs: true,
};

And yet 895 files are still saved to snapshots. What am I doing wrong? Or misunderstanding?

**Comment by EdouardDem** [comment]
Hi @Dithilli 
In the `snapshot` folder, you get the schema of Directus (i.e. the database structure and the models configurations)
In the `collections` folder, you get the data of the data from the built-in collections (flows, settings, operations, etc.)

If you want to ignore the snapshot, set `snapshot` to `false`.
By the way, if you want to get only the `flows`, you need the include also the `operations`. `flows` depends on `operations`.
